### PR TITLE
Added Support to generate standup notes in markdown 

### PR DESCRIPTION
**Pull request by darshkpatel**
Example: 
```
 **Yesterday**:
    - Worked on PR [Static user data fetch script and dump #6](https://github.com/bundly/dash/pull/6) (Status: merged)
    - Worked on PR [Linter config #5](https://github.com/bundly/dash/pull/5) (Status: merged)
    - Worked on PR [Basic Github Notification Feed  #4](https://github.com/bundly/dash/pull/4) (Status: merged)
    - Pushed 13 Commits to [bundly/dash](https://github.com/bundly/dash)
    - Pushed 1 Commits to [vercel/next.js](https://github.com/vercel/next.js)
    - Pushed 1 Commits to [darshkpatel/GitTest](https://github.com/darshkpatel/GitTest)
    - Opened Issue [Dashboard UI #3](https://github.com/bundly/dash/issues/3)
    - Opened Issue [Make Github Activity Component #2](https://github.com/bundly/dash/issues/2)
    - Opened Issue [Setup OAuth  #1](https://github.com/bundly/dash/issues/1)
    - Reviewed PR [WIP Setup OAuth #7](https://github.com/bundly/dash/pull/7)
    - Reviewed 1 Pull Requests
    - 5 Comments on Issue Discussions
```

Also spent quite a while to write an optimized graphql query

### CODE REVIEW

**Comment by sauravhiremath** [comment]
Will look into this tomorrow

**Comment by darshkpatel** [comment]
@sauravhiremath This is complete have a look so i can merge

File: src/scripts/notesGenerator.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+function getCommentCount(issueComments, currentTime) {
+  // Filter All comments from the past 24 hours
+  const targetTime = new Date(currentTime);
+  targetTime.setHours(targetTime.getHours() - 24);
+  return issueComments.nodes.filter((comment) => {
+    const commentTime = new Date(comment.updatedAt);
+    return commentTime > targetTime;
+  }).length;
+}
+
```
Comment (sauravhiremath) [inline src/scripts/notesGenerator.js:12]: Could you add `getTotalCommits()` count after the Fellowship started? Coz we are adding a leader-board, hopefully

**Review by sauravhiremath — CHANGES_REQUESTED** [review]
Looks solid 🎉. Just needs another function, as mentioned.

Can merge after that!

File: src/scripts/notesGenerator.js (hunk 1 above)
Comment (darshkpatel) [reply src/scripts/notesGenerator.js:12]: Will add separately, needs an entirely different query

**Pull request merged**